### PR TITLE
dbeaver/pro#2674 [TE] Shared credentials button is not visible on small screens

### PR DESCRIPTION
**Pull request by serjiokov**
Scrolled composite added

### CODE REVIEW

**Comment by serjiokov** [comment]
https://github.com/dbeaver/dbeaver/assets/32031248/ac1d5910-e9be-4640-b755-63ed1acb9927

**Comment by E1izabeth** [comment]
Why these "Save credentials" checkbox and "Manage" button are so far from Authentication method combo?

**Comment by ShadelessFox** [comment]
Perhaps because of the "Save credentials" checkbox. I wouldn't say I like it either, but fixing it wouldn't be that trivial.

File: plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionPageSettings.java [hunk 1]
```diff
@@ -425,9 +422,13 @@ private CTabItem createPageTab(@NotNull IDialogPage page, int index) {
         item.setToolTipText(page.getDescription());
 
         if (page.getControl() == null) {
-            final Composite placeholder = new Composite(tabFolder, SWT.NONE);
+            final ScrolledComposite sc = new ScrolledComposite(tabFolder, SWT.BORDER | SWT.H_SCROLL);
+            sc.setExpandHorizontal(true);
+            sc.setExpandVertical(true);
+            final Composite placeholder = new Composite(sc, SWT.NONE);
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionPageSettings.java]: I don't think you need a placeholder here. Put controls directly inside your `ScrolledComposite`

Comment (serjiokov) [reply plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/dialogs/connection/ConnectionPageSettings.java]: Just for use 'setContent' - require to specify scrollable content

**Review by E1izabeth — APPROVED** [review]

**Review by ShadelessFox — APPROVED** [review]

**Pull request merged**